### DISCUSSION TITLE
PlexTraktSync and Plex Meta Manager at once?

### DISCUSSION THREAD

**Discussion by DreamStatic** [Taxel/PlexTraktSync#317] [Q&A] [answered]
I am curious if anyone has had any luck running this and Plex Meta Manager together? It seems they are using conflicting Trakt libraries. When I install the requirements for each it breaks the other. I tried to just use the Plex Meta Manager Trakt requirement, but PlexTraktSync doesn't appear to work doing this. I am new to both projects so maybe something easy I am missing. Thanks for the info.

**Comment by glensc** [comment, marked as answer]
this will not work, you need to virtualize dependencies (docker, virtualenv, pipenv).

closing as duplicate of https://github.com/Taxel/PlexTraktSync/issues/306

here are the details:
- https://github.com/Taxel/PlexTraktSync/issues/306#issuecomment-830217795
- https://github.com/meisnate12/Plex-Meta-Manager/blob/v1.9.2/requirements.txt#L5
- https://github.com/Taxel/PlexTraktSync/blob/0.9.15/requirements.txt#L6

**Reply by DreamStatic** [reply]
Thank you for the info.
